spec.bs

```diff
@@ -543,6 +541,23 @@ Add the following to the [=registry of standardized payment methods=] in
     : "[=secure-payment-confirmation=]"
     :: The <a href="https://w3c.github.io/secure-payment-confirmation/">Secure Payment Confirmation</a> specification.
 
+### Modification of user activation requirement ### {#sctn-modify-user-activation-requirement}
+
+In the steps for when {{PaymentRequest/show|PaymentRequest.show()}} is called,
+modify steps 2 and 3:
+
+2. If the [=relevant global object=] of [=request=] does not have
+        [=transient activation=], the user agent MAY:
+
+    1. Return [=a promise rejected with=] with a {{"SecurityError"}}
+            {{DOMException}}.
+
+3. Otherwise, [=Consume user activation=] of the [=relevant global object=].
+
+NOTE: This change allows the user agent to decide to skip the user activation
+      requirement. See [[#sctn-security-user-activation-requirement]] for
+      security considerations with this change.
```

Comment:
```suggestion
      requirement, for example to support redirect authentication flows where a
      user activation may not be present upon redirect. See
      [[#sctn-security-user-activation-requirement]] for security considerations
      with this change.
```

(may not be formatted to 80 lines correctly)

I think its worth trying to explain why we modify this, either in this note as I suggest here, or in the main text on line 546 (I have no preference; @ianbjacobs ?)